src/webview/webview2/mod.rs

```diff
@@ -804,17 +804,30 @@ window.addEventListener('mousemove', (e) => window.chrome.webview.postMessage('_
     )
   }
 
-  fn execute_script(webview: &ICoreWebView2, js: String) -> windows::core::Result<()> {
+  fn execute_script(
+    webview: &ICoreWebView2,
+    js: String,
+    callback: impl FnOnce(String) + Send + 'static,
+  ) -> windows::core::Result<()> {
     unsafe {
       webview.ExecuteScript(
         PCWSTR::from_raw(encode_wide(js).as_ptr()),
-        &ExecuteScriptCompletedHandler::create(Box::new(|_, _| (Ok(())))),
+        &ExecuteScriptCompletedHandler::create(Box::new(|_, return_str| {
+          let json_value: serde_json::Value = serde_json::from_str(&return_str).unwrap();
```

Comment:
It seems return_str is already json. Do we need to convert again?

Reply:
No, I just tested it again, and it's unnecessary, I'll remove it.